que.go

```diff
@@ -168,55 +169,38 @@ func (c *Client) Enqueue(j *Job, txn pgxV5.Tx) error {
 //
 // It is the caller's responsibility to Commit or Rollback the transaction after
 // this function is called.
-func (c *Client) EnqueueInTx(j *Job, txn pgxV5.Tx) error {
+func (c *Client) EnqueueInTx(j *Job, txn queryable) error {
 	return execEnqueue(j, txn)
 }
 
-func execEnqueue(j *Job, txn pgxV5.Tx) error {
+func execEnqueue(j *Job, txn queryable) error {
 	if j.Type == "" {
 		return ErrMissingType
 	}
 
 	queue := &pgtype.Text{
 		String: j.Queue,
-		Status: pgtype.Null,
-	}
-	if j.Queue != "" {
-		queue.Status = pgtype.Present
+		Valid:  j.Queue != "",
```

Comment:
Wouldn't `false` be more correct here?

I'm slightly concerned, possibly being overcautious, about changes in behavior in this function.

Reply:
for all of those properties we had the following snippet before:
```
	queue := &pgtype.Text{
		String: j.Queue,
		Status: pgtype.Null,
	}
	if j.Queue != "" {
		queue.Status = pgtype.Present
	}
```
In the first step we create the new variable with the value and `status=null` and then as a second step if there was a value then we set the status to present.
This was necessary because there is no tenary operator in go. But the new update of pgx replaced the status-enum with a boolean, so instead of
```
	queue := &pgtype.Text{
		String: j.Queue,
		Valid: false,
	}
	if j.Queue != "" {
		queue.Valid = true
	}
```
we can condense it into
```
	queue := &pgtype.Text{
		String: j.Queue,
		Valid: j.Queue != "",
	}
```